### DISCUSSION TITLE
The new mobile web experience is much slower and less touch friendly than the classic version

### DISCUSSION THREAD

**Discussion by Pentaphon** [standardnotes/forum#2571] [General]
This isn't a specific bug report but more of a general complaint. 

I opted into the new mobile web experience for the Android app and I was shocked at how slow it was compared to the native app. I tried launching it 5 times and it was just as slow each time to open.  This is the slowest app I've ever used and even the classic app wasn't that fast to begin with. I can't believe how slow it was to launch.  I can't imagine how slow it would be on a lower end phone. 

Second, the new mobile web experience for Android is less touch-friendly than before. Everything seems smaller to touch and doesn't scale as well for touch as the classic app did. The menus are just not as intuitive, either. I kept opening the note options by mistake at first. 

Third, the text legibility on Android is horrendous.  It's much harder to read the default text on the new mobile web experience. All the text is too thin, faded and just not suitable for small screens. It used to be clear, crisp and bold but now it just seems devoid of any clarity and sharpness. 

I tried using the new mobile experience for about 10 minutes and I went right back to the classic experience. I hope there's way more improvements to be made because this is a huge step back. It feels like I'm using a bad web wrapper and I really do not like it.

**Comment by Betadawg** [comment]
I also had issues with the text ligibility and found the font (the original font, not monospace) to be too small and thin. 

Agree with the icons being too small and the menus being unintuitive.

I would like to add one issue that is the inability to swipe out of a note (closes the app) and return to the main page of notes. Instead I have to stretch my finger to the top left corner and click to exit a note.

Also the additional buttons to add an attachment or a tag take up too much space and shifts a note's content down.

At the very least I hope there will be an option for us that prefer the classic experience to stick with it.

**Comment by fastjack77** [comment]
Writing a note with Standard Notes is _extremely_ slow on my cell phone - there is a pause for every character written.

- The problem is intermittant; while writing the same note, the speed of writing will vary. The same happens when using backspace (as if an app was checking and waiting for data for every stroke).
- Outside the action of writing/modifying a note, the application runs smoothly.
- This was tested on Standard Notes v3.144.0.
- The account was synced.
- I'm using Android 13 on a Samsung A52s.
- Except for this specific issue with Standard Notes, all applications run very fast.
- No virus or malwares detected using an updated popular antivirus.
- No other obvious issues noticed from the OS or the cell phone.

**Comment by effieeee** [comment]
Hi @fastjack77, I may need to troubleshoot this with you via email, so please reach out to us at help@standardnotes.com when you can! 🙏
